charts/victoria-metrics-agent/values.yaml

```diff
@@ -90,6 +90,9 @@ env:
   #       name: auth_secret
   #       key: password
 
+# extra Labels for Pods, Deployment and Statefulset
+extraLabels: {}
```

Comment:
Hi we need to separate labels for two
- extraLabels - these are labels for sts, deploy
- podLabels - these are labels for pod under sts, deploy

Reply:
sure, I can do that 🆗 I was inspired from [vmcluster](https://github.com/VictoriaMetrics/helm-charts/blob/master/charts/victoria-metrics-cluster/templates/vmselect-statefulset.yaml#L11) chart which has `extraLabels` as common value for both `Pods` and `Statefulset`